frontend/app-development/router/routes.tsx

```diff
@@ -37,7 +37,7 @@ const isLatestFrontendVersion = (version: AppVersion): boolean =>
 const UiEditor = () => {
   const { org, app } = useStudioUrlParams();
   const { data } = useAppVersionQuery(org, app);
-  return isLatestFrontendVersion(data) ? <UiEditorLatest /> : <UiEditorV3 />;
+  return isLatestFrontendVersion(data) ? <UiEditorLatest /> : <UiEditorLatest />;
```

Comment:
This should be reverted to UiEditorV3. 😊 
```suggestion
  return isLatestFrontendVersion(data) ? <UiEditorLatest /> : <UiEditorV3/>;
```